platforms/hyperledger-besu/configuration/reset-network.yaml

```diff
@@ -121,7 +162,8 @@
       vars:
         kubernetes: "{{ item.k8s }}"
       loop: "{{ network['organizations'] }}"
-      ignore_errors: yes
+      when: patch_result is succeeded
```

Comment:
Why do we have this `when` condition?
(Its fine to put the condition considering the other case `when: !patch_result is succeeded` is also tested or covered)